lua/neotest-go/init.lua

```diff
@@ -181,6 +187,28 @@ function adapter.discover_positions(path)
       arguments: (argument_list . (interpreted_string_literal) @test.name))
       @test.definition
   ]]
+
+  if get_experimental_opts().test_table then
+    query = query .. [[
+
+    (short_var_declaration
```

Comment:
I considered something like this, but I really worry that this query will be very brittle and will essentially never be able to cover basically any variation. If a user calls the test cases just `cases` then it won't work. If their project doesn't call the fields `name` or `desc` this also won't work.

Maybe these things could be configurable, though?